src/display/process/procGraphs.go

```diff
@@ -96,9 +98,18 @@ func ProcVisuals(endChannel chan os.Signal,
 
 		// Adjust Grid dimensions
 		myPage.Grid.SetRect(0, 0, w, h)
-		ui.Render(myPage.Grid)
+		help.Resize(w, h)
+		ui.Clear()
+		if helpVisible {
+			ui.Render(help)
+		} else {
+			ui.Render(myPage.Grid)
+		}
+
 	}
 
+	updateUI()
```

Comment:
This breaks the UI as the PID table rows in the grid is not initialized. Removing this should fix the issue.

Reply:
Done. Is that required in the other two graphs though, if it can be removed here?

Reply:
I think it is, as it helps render an empty UI at first thus reducing lags.